clients/ts/signalr/src/ServerSentEventsTransport.ts

```diff
@@ -38,18 +38,20 @@ export class ServerSentEventsTransport implements ITransport {
 
         this.logger.log(LogLevel.Trace, "(SSE transport) Connecting");
 
+        this.url = url;
```

Comment:
Can you put a comment in about why we do this before updating the URL with the query string? (Because `this.url` is only used by the send, which can set headers and thus doesn't need the query string goop)